.gitignore

```diff
@@ -19,6 +19,7 @@ coverage/*
 
 # Dist assets
 dashboard/assets/dist
+dashboard/apps/prototype/static/dist/
```

Comment:
this line shouldn't be on ignore file. it's not a directory generated by the build any more. it's better you just remove the directory from your local file system. in addition, it's a good practice to have a `~/.gitignore_global` file, something like this http://augustl.com/blog/2009/global_gitignores/